docs/medical-api/fhir/data-types/extension.mdx

```diff
@@ -0,0 +1,13 @@
+| value[x]  | No       | base64Binary, boolean, canonical, code, date, dateTime, decimal, id, instant, integer, markdown, oid, positiveInt, string, time, unsignedInt, uri, url, uuid, [Address](/medical-api/fhir/data-types/address), [Age](/medical-api/fhir/data-types/age), [Annotation](/medical-api/fhir/data-types/annotation), [Attachment](/medical-api/fhir/data-types/attachment), [CodeableConcept](/medical-api/fhir/data-types/codeableconcept), [Coding](/medical-api/fhir/data-types/coding), [ContactPoint](/medical-api/fhir/data-types/contactpoint), [Count](/medical-api/fhir/data-types/count), [Distance](/medical-api/fhir/data-types/distance), [Duration](/medical-api/fhir/data-types/duration), [HumanName](/medical-api/fhir/data-types/humanname), [Identifier](/medical-api/fhir/data-types/identifier), [Money](/medical-api/fhir/data-types/money), [Period](/medical-api/fhir/data-types/period), [Quantity](/medical-api/fhir/data-types/quantity), [Range](/medical-api/fhir/data-types/range), [Ratio](/medical-api/fhir/data-types/ratio), Reference&lt;&gt;, [SampledData](/medical-api/fhir/data-types/sampleddata), [Signature](/medical-api/fhir/data-types/signature), [Timing](/medical-api/fhir/data-types/timing), [ContactDetail](/medical-api/fhir/data-types/contactdetail), [Contributor](/medical-api/fhir/data-types/contributor), [DataRequirement](/medical-api/fhir/data-types/datarequirement), [Expression](/medical-api/fhir/data-types/expression), [ParameterDefinition](/medical-api/fhir/data-types/parameterdefinition), [RelatedArtifact](/medical-api/fhir/data-types/relatedartifact), [TriggerDefinition](/medical-api/fhir/data-types/triggerdefinition), [UsageContext](/medical-api/fhir/data-types/usagecontext), [Dosage](/medical-api/fhir/data-types/dosage), [Meta](/medical-api/fhir/data-types/meta) | Value of extension                            |
```

Comment:
It seems we're using the property `short` of FHIR's definition, I wonder if we could use `definition` instead - see how much more context it gives

<img width="1042" alt="image" src="https://github.com/metriport/metriport/assets/2132564/c576283d-b67e-4aa3-9063-455a9d8d5497">

NABD for now, though

Reply:
I went with the short one because i felt like the long one in markdown would end up being overwhelming but if we want to provide more context i can add the regular one

Reply:
I haven't checked all resources/data types... maybe if we add the link to the FHIR spec we can keep the short one?